github/table_github_audit_log.go

```diff
@@ -0,0 +1,146 @@
+package github
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/go-github/v45/github"
+	pb "github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableGitHubAuditLog(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_audit_log",
+		Description: "Gets the audit log for an organization.",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "organization", Require: plugin.Required},
+				{Name: "phrase", Require: plugin.Optional},
+				{Name: "include", Require: plugin.Optional},
+				{Name: "action", Require: plugin.Optional},
+				{Name: "actor", Require: plugin.Optional},
+				{Name: "created_at", Require: plugin.Optional, Operators: []string{">", ">=", "<", "<=", "="}},
+			},
+			Hydrate: tableGitHubAuditLogList,
+		},
+		Columns: []*plugin.Column{
+			{Name: "organization", Type: pb.ColumnType_STRING, Transform: transform.FromQual("organization")},
+			{Name: "phrase", Type: pb.ColumnType_STRING, Transform: transform.FromQual("phrase"), Description: "The search phrase for your audit events."},
+			{Name: "include", Type: pb.ColumnType_STRING, Transform: transform.FromQual("include"), Description: "The event types to include: web, git, all."},
+
+			// Top columns.
+			{Name: "id", Type: pb.ColumnType_STRING, Description: "The id of the audit event.", Transform: transform.FromField("DocumentID")},
```

Comment:
What does a document ID look like? And how common is that terminology in GitHub's UI, docs, etc.?

When column names deviate from property names the source uses, e.g., using `id` instead of `document_id`, there's usually a clear reason behind it. So in this case, if most people refer to audit logs' document IDs, if we change that name here, it may confuse some users if they're expecting a column called `document_id`.